Gemfile

```diff
@@ -10,9 +10,13 @@ gem 'byebug', :platform => :ruby
 gem 'ruby-debug', :platform => :jruby
 gem 'rack', '1.6.0'
 
+gem 'dotenv'
+
 gem 'mysql2', :platform => :ruby
 gem 'pg', :platform => :ruby
 
+gem 'activerecord', '~> 4.2.6'
```

Comment:
In order to confirm the test against Procore, I wanted to make sure that this gem was testing against the same version of AR, so I made it identical to Procore's current version.

Once we get past the integration testing, and certainly before creating a PR back to the originating repo, I'll remove the specific AR version.